avm/res/app/job/main.bicep

```
@@ -30,19 +69,19 @@ param roleAssignments roleAssignmentType
 param enableTelemetry bool = true
 
 @description('Required. List of container definitions for the Container App.')
-param containers array
+param containers containerType
 
 @description('Optional. List of specialized containers that run before app containers.')
-param initContainersTemplate array?
+param initContainersTemplate initContainerType?
```

Since at it, probably in this case it's the parameter name which should align to the api. What do you think?
```suggestion
param initContainers initContainersType?
```

---

I agree. It looks and feels cleaner.

---

It would make sense to adjust the naming of the container-app module as well.